escoba/src/__store.js

```diff
@@ -0,0 +1,293 @@
+import Vue from 'vue'
```

Comment:
This file (and all the .vue files) seem out of scope for this PR ? what was the context of this change?